lakers-c/src/lib.rs

```diff
@@ -26,26 +26,32 @@ static HEAP: Heap = Heap::empty();
 pub struct EADItemC {
     pub label: u16,
     pub is_critical: bool,
+    /// The value is only emitted if this is true (otherwise it is an EAD item is just a label)
```

Comment:
do you mean: otherwise it is *NOT* an EAD item (?)

Reply:
```suggestion
    /// The value is only emitted if this is true (otherwise it is an EAD item that has just a label)
```

It's still an EAD item, but the wording was wrong.